src/style.css

```diff
@@ -0,0 +1,136 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
```

Comment:
Good job! 
But you should create separated files for BEM blocks. Checklist P.11.
It'll be better if you will add two different files for .card and .stars blocks. Also you can create a separated file to reset your margins and set default styles like font.

Reply:
🔥